### ISSUE TITLE
mearn stack

### ISSUE THREAD

**Issue by manikanta290** [remy/nodemon#2227]
nodemon] app crashed waiting for file changes before starting...

[nodemon) restarting due to changes... [nodemon] starting node server.js

(node:15228) [MONGODB DRIVER] Warning: useNewUrlParser is a deprecated option: useNewUrlParser has no effect since Node.js Driver version 4. 0.0 and will be removed in the next major version

(Use node --trace-warnings ... to show where the warning was created)

(node: 15228) [MONGODB DRIVER] Warning: useUnifiedTopology is a deprecated option: useUnifiedTopology has no effect since Node.js Driver vers ion 4.0.0 and will be removed in the next major version

Example app listening on port 3000

MongoDB connection FAIL

[nodemon] app crashed waiting for file changes before starting...

**Comment by remy** [comment]
What's the actual issue here?

**Comment by mohit-nagaraj** [comment]
hey idts this is issue from nodemon side rather ur mongo db driver, it does state that useNewUrlParser & useUnifiedTopology is deprecated. dont use them

**Comment by ravin00** [comment]
> hey idts this is issue from nodemon side rather ur mongo db driver, it does state that useNewUrlParser & useUnifiedTopology is deprecated. dont use them

i agree with this this happened to me to since theses are deprecated you can use 

Set mongoose.set('useNewUrlParser', true)
Set mongoose.set('useFindAndModify', false)
Set mongoose.set('useCreateIndex', true)
Set mongoose.set('useUnifiedTopology', true)
Replace update() with updateOne(), updateMany(), or replaceOne()
Replace remove() with deleteOne() or deleteMany()
Replace count() with countDocuments()

you can use this if you get any deprecated warnings

**Comment by remy** [comment]
So… nothing to do with nodemon?

**Comment by ravin00** [comment]
> So… nothing to do with nodemon?

I think no this should work
